ui/src/datatypes.vue

```diff
@@ -47,7 +54,7 @@ import pageheader from '@/components/pageheader'
 import contact from '@/components/contact'
 import app from '@/components/app'
 import tags from '@/components/tags'
-
+let query_debounce;
```

Comment:
let's use cameCasing for all new code. So do `let queryDebounce`. Eventually we will update all variables with old_naming_schema.